frontend/common/data/base/_data.js

```diff
@@ -74,8 +74,8 @@ module.exports = {
     return this._request('delete', url, data, headers)
   },
 
-  get(url, data, headers) {
-    return this._request('get', url, data || null, headers)
+  get(url, data, headers, isExternal = false) {
```

Comment:
I think I mentioned this in another PR, but we can just base this on if the url is Project.api. Also at request(url not just get

Reply:
Done.